release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3322,7 +3322,52 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
-// 4.14.29
+[id="ocp-4-14-30"]
+=== RHSA-2024:3881 - {product-title} 4.14.30 bug fix and security updates
+
+Issued: 2024-06-19
+
+{product-title} release 4.14.30, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:3881[RHSA-2024:3881] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:3918[RHSA-2024:3918] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.14.30 --pullspecs
+----
+
+[id="ocp-4-14-30-bug-fixes"]
+==== Bug fixes
+
+* Previously,  (link:https://issues.redhat.com/browse/OCPBUGS-35085[*OCPBUGS-35085*])
```

Comment:
* Previously, the OpenShift Agent Installer reported installed SATA SDDs as removable and refused to use any of them as installation targets. With this release removable disks are eligible for installation and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-35085[*OCPBUGS-35085*])